src/Symfony/Component/Notifier/Transport/Transports.php

```diff
@@ -71,8 +69,8 @@ public function send(MessageInterface $message): void
 
         if (!$this->transports[$transport]->supports($message)) {
             throw new LogicException(sprintf('The "%s" transport does not support the given message.', $transport));
-        }
+	}
```

Comment:
Looks like it's a wrong change here.

Reply:
Yes this is strange. It is a tab (I may have changed the file with another editor). I fix it. Good catch!